cms/static/cms/js/require-config.js

```diff
@@ -300,6 +300,9 @@
                        "rangeslider", "share-annotator", "richText-annotator", "reply-annotator",
                        "tags-annotator", "flagging-annotator", "grouping-annotator", "diacritic-annotator",
                        "openseadragon", "jquery-Watch", "catch", "handlebars", "URI"]
+            },
+            "draggabilly": {
```

Comment:
I don't think you need this in CMS (the shim part) because CMS uses RequireJS throughout.